api/v1alpha1/project_config_types.go

```diff
@@ -134,7 +134,7 @@ type GitLabWebhookReceiverConfig struct {
 	// For cluster-scoped webhook receivers, the referenced Secret must be in the
 	// designated "cluster Secrets" namespace.
 	//
-	// The secret is expected to contain a `gitlab-secret` key containing the
+	// The secret is expected to contain a `secret-token` key containing the
```

Comment:
👍 Appears consistent with the form:

![Screenshot 2025-06-09 at 3 12 59 PM](https://github.com/user-attachments/assets/09efb59f-cdd2-4653-94b6-109d45c5f3c5)